src/Console/Application.php

```diff
@@ -162,4 +163,15 @@ public function afterRun($callable)
     {
         $this->after = $callable;
     }
+
+    public function renderException($e, $output)
```

Comment:
hmm depeding on the version of symfony console we use (prefer-stable vs. prefer-lowest) php expects a different signature for this method... not sure how to handle this case...

Reply:
looking at it from a different angle. does it actually make sense that we CI build deployer with php 7.0/7.1/7.2 and prefer-stable vs. prefer-lowser, as the deployer binary/executable itself will finally be bundled with whatever the build process will decide?

it will always contain what prefer-latest will suggest and noone will use this repos in a prefer-lowest fashion, right?

Reply:
Actually a lot of people use it as dev dependency of composer. And we need to support all of them. Mybe there is a better place to handle this logging. See CommandTask.